content/community/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Community"
+headline: "Get Involved"
+tagline: ECD Tools is a community of members, adopters, developers, and other contributors. Join us!
+date: 2020-09-30T14:53:00+02:00
+hide_page_title: true
+hide_sidebar: true
+show_featured_footer: false
+container: "container-fluid"
+---
+{{< grid/section-container>}}
```

Comment:
You can remove these 2 lines here, the two closing tags at the bottom and line 9: container: "container-fluid".

Reply:
But then that would remove the top jumbotron, right?